### PR TITLE
Play and summarize Artsy podcasts.

### LINKED ISSUE #51: Ask Artsy to play the latest artsy podcast
Ask Artsy to play the artsy podcast 3
etc.

Probably through SoundCloud [Api](https://developers.soundcloud.com/docs/api/reference), https://soundcloud.com/artsypodcast.

### PR DESCRIPTION
**Pull request by dblock**
Closes #51.

### CODE REVIEW

File: functions/artsy/index.js [hunk 1]
```diff
@@ -276,6 +299,83 @@ app.intent('ShowsIntent', {
...
+    }
+
+    podcastStream.then(function(audioMpegEnclosure) {
+      var streamUrl = audioMpegEnclosure.url.replace('http://', 'https://'); // SSL required by Amazon, available on SoundCloud
+      var stream = {
+        url: streamUrl,
+        token: streamUrl,
+        offsetInMilliseconds: 0
+      }
+      res.audioPlayerPlayStream('REPLACE_ALL', stream);
```
Comment (ashkan18) [inline functions/artsy/index.js]: magic?! Need to read about `audoPlayerPlayStream`

File: functions/artsy/models/podcast.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+var Q = require('q');
+var feedparser = require('feedparser-promised');
+var _ = require('underscore');
+_.mixin(require('underscore.string').exports());
+
+function Podcast() {
+  this.rssUrl = 'https://feeds.soundcloud.com/users/soundcloud:users:211089382/sounds.rss';
```
Comment (ashkan18) [inline functions/artsy/models/podcast.js]: I was going to suggest make this an `env` but I guess there is no point if we lose `env` everytime we deploy.

**Review by ashkan18 — APPROVED** [review]
Super cooooool!

File: functions/artsy/index.js (hunk 1 above)
Comment (dblock) [inline functions/artsy/index.js]: Yes, this is a "native" function of the API of alexa-app, see https://github.com/alexa-js/alexa-app

File: functions/artsy/models/podcast.js (hunk 2 above)
Comment (dblock) [inline functions/artsy/models/podcast.js]: Yeah, and it's not likely to change without having to do code changes.

**Pull request merged**